### DISCUSSION TITLE
Mise en production - delivery final

### DISCUSSION THREAD

**Discussion by longuebrayj** [adamvogier/cmdb#19] [General]
Hello,
@Molzonas @FlexReflx @McBaxck @pipeau 

Pour faire suite à notre échange sur la finalité du projet, voici un compte rendu de ce qui a été dit : 

Il a été décidé une deadline à 15H00 
**Mise en prod à 14H00.** (Memo : ajouter un tag V1.0 sur le commit de merge de MEP)
On réalisera la rétrospective dans la foulée. 

La synthèse du projet est dispo dans le wiki. 
J'ai pré set up la rétro. L'idée est dans un premier temps de valider les acquis puis d'effectuer un RETEX. 
Si vous avez d'autres idées, n'hésitez pas 💯

**Comment by adamvogier** [comment]
OK pour moi à 14H00

**Comment by FlexReflx** [comment]
OK pour 14h00, il faudrait commencer par la base de données !

**Comment by pipeau** [comment]
ok pour moi bon courage

**Comment by Molzonas** [comment]
OK pour la mise en production à 14H00.

Il sera nécessaire de faire les vérifications sur les parties **classe d'entités**, **classes de liaisons avec la base de données**, **classes REST**, **fonctions et méthodes de démarrage du serveur REST**, **tests unitaires**, **tests d'intégration** et **mise de commentaires**.

Le minimum vital pour terminer le projet correctement, pour 14H00, serait:
* Base de données fonctionnelle et utilisable
* Classes des entités terminées et vérifiées
* Classes de liaisons terminées et vérifiées
* Ouverture du REST sur le réseau local
* Récupération de données de test depuis le réseau local (ex: `http://localhost:8080/v1/servers/1` récupère le serveur à l'ID 1 depuis la base de données)
* Commentaire complet et précis du code
* Tests unitaires
* Tests d'intégration

En travaillant bien et en se répartissant les tâches, nous serons bons.

Voir avancement de #12 #9 (@FlexReflx @pipeau), #14 #13 (@McBaxck @Molzonas ), #10 #11 (@longuebrayj ).

**Comment by longuebrayj** [comment]
pour la répartition des tâches, se référer aux attributions des issues et CR des "daily" dans le wiki qui sont davantage proches de la réalité
